pkgs/applications/science/electronics/nxpmicro-mfgtools/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv,
+  fetchFromGitHub,
+  cmake,
+  bzip2,
+  libusb1,
+  libzip,
+  openssl,
+  pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname   = "nxpmicro-mfgtools";
```

Comment:
Maybe this should be just be called `mfgtools`? Not 100% sure though ;).

Reply:
Hi, thanks for all the feedback!

I am not sure which way's better.

Usually I would add the "nxpmicro-" as I don't know how many mfgtools can pop up in the future (doubt there will be any more), with the assumption that adding "namespaces" later would be harder. I am really tempted to make it only "mfgtools" as it's less text to write.

Your call. I can amend the commit based on what you decide.

Reply:
Thanks for the changes. There was a small syntax error that caused evaluation to fail, so I fixed that. I also used `nixpkgs-fmt` to format the derivation.

I think the prefix may be better here. As you say, `mfgtools` is very generic. I don't think there will be another package with the same name, but with the prefix it's easier to recognize and quickly find.

Reply:
Whoops, seems like we are force-pushing at the same time. If you only fixed the syntax error, I can push my changes again.